dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/PasswordNullifier.java

```diff
@@ -19,25 +19,38 @@
     /** Search patterns to use for passwords */
     private static final String PASSWORD_PATTERN = "password=";
     private static final String CLIENT_SECRET_PATTERN = "client_secret=";
+    private static final String BASICAUTH_PATTERN = "basic ";
```

Comment:
Should this be lower cased? If the header value contains "Basic " and assuming the source.indexOf(pattern) invocation at line 107 is case-sensitive, wouldn't that be problematic?